src/components/sectionList/modelValue/ModelValue.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import { Schema, SchemaFieldProperty } from '../../../lib'
+import { ModelValueRenderer } from './ModelValueRenderer'
+
+export type ValueDetails = {
+    schemaProperty: SchemaFieldProperty
+    value: unknown
+}
+
+type ModelValueProps = {
+    schema: Schema
+    modelPropertyName: string
+    value: unknown
+}
+
+export const ModelValue = ({
+    schema,
+    modelPropertyName,
+    value,
+}: ModelValueProps) => {
+    const schemaProperty = schema.properties[modelPropertyName]
+
+    if (!schemaProperty) {
+        console.warn(
+            `Property ${modelPropertyName} not found in schema, value not rendered: ${value}`
+        )
+        return null
+    }
+
+    return (
+        <span>
+            <ModelValueRenderer value={value} schemaProperty={schemaProperty} />
```

Comment:
The "ModelValueRenderer"-logic could've been in this `ModelValue`-component. However I think it's more flexible to have it in a separate one, since it's easier to style and add common logic in this one - since `ModelValueRenderer` returns early.